instrumentation/sinatra/lib/opentelemetry/instrumentation/sinatra/instrumentation.rb

```diff
@@ -13,6 +13,8 @@ module Sinatra
       # instrumentation
       class Instrumentation < OpenTelemetry::Instrumentation::Base
         install do |_|
+          OpenTelemetry::Instrumentation::Rack::Instrumentation.instance.install({})
```

Comment:
The end user still has to manually add the middleware correct? 
Is the Sinatra middleware stack independent of the Rack stack? will this always ensure the middleware is installed after the Rack middleware is installed?

Reply:
NVM I missed the injection int eh extensions file.